Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -72,7 +72,7 @@ class SearchViewController: SiteTableViewController, KeyboardHelperDelegate, Loa
     private let suggestionCell = SuggestionCell(style: UITableViewCellStyle.Default, reuseIdentifier: nil)
 
     private var suggestionPrompt: UIView?
-
+    private let faviconFetcher = FaviconFetcher()
```

Comment:
How come you need an instance of FaviconFetcher when you're using it to call a `class` method?